### ISSUE TITLE
aiohttp.Timeout is now gone (as of aiohttp 3.0.0)

### ISSUE THREAD

**Issue by markgajdosik** [mvantellingen/python-zeep#698]
Hello, zeep 2.5 with asynchronous transport fails with aiohttp >= 3.0.0 as `aiohttp.Timeout` has been deprecated (see https://github.com/aio-libs/aiohttp/issues/2348). According to the ticket, zeep might need to use `async_timeout.timeout` context manager instead of relying on the deprecated `aiohttp.Timeout`.

**Comment by ba1dr** [comment]
Quick workaround while it is not fixed:

```
import aiohttp
import async_timeout
aiohttp.Timeout = async_timeout.timeout
from zeep import Client as ZeepClient
```

**Comment by mvantellingen** [comment]
Fixed in #735 i think
